src/main/java/com/ksm/wstbackoffice/validation/ValidationConstant.java

```diff
@@ -0,0 +1,5 @@
+package com.ksm.wstbackoffice.validation;
+
+public class ValidationConstant {
```

Comment:
The best practice is: constants if not possible to put in the class they are going to be used, should be placed in the noninstantiable utility class.
Please google how to create noninstantiable class.
Here is an example of it: https://stackoverflow.com/questions/8848107/how-to-construct-a-non-instantiable-and-non-inheritable-class-in-java

Reply:
done